samples/wasi-threads/wasm-apps/wasi_thread_start.S

```diff
@@ -0,0 +1,20 @@
+	.globaltype	__stack_pointer, i32
+	.functype	__wasi_thread_start_C (i32, i32) -> ()
+
+	.globl	wasi_thread_start
+
+wasi_thread_start:
```

Comment:
if this file is a modified copy from wasi-libc, please mention it.

Reply:
updated